### PR TITLE
ddl: Export some variables and remove useless tests

### PR DESCRIPTION
**Pull request by zimulala**
Move the unit test to [PR](https://github.com/pingcap/tidb-test/pull/309).
Fix a bug when new a session.

### CODE REVIEW

**Comment by zimulala** [comment]
PTAL @tiancaiamao @shenli @hanfei1991 @coocood

**Comment by coocood** [comment]
LGTM

File: ddl/owner_manager.go [hunk 1]
```diff
@@ -46,8 +46,10 @@ type OwnerManager interface {
 }
 
 const (
-	ddlOwnerKey               = "/tidb/ddl/owner"
-	bgOwnerKey                = "/tidb/ddl/bg/owner"
+	// DDLOwnerKey is exported for testing.
```
Comment (shenli) [inline ddl/owner_manager.go]: The comment should describe what is this variable mean.

File: ddl/owner_manager.go [hunk 2]
```diff
@@ -174,29 +178,38 @@ func (m *ownerManager) campaignLoop(ctx goctx.Context, key string, wg *sync.Wait
...
+		m.setOwnerVal(key, true)
 
-		m.watchOwner(ctx, string(resp.Kvs[0].Key))
+		m.watchOwner(ctx, ownerKey)
 		m.setOwnerVal(key, false)
 	}
 }
 
+// GetOwnerInfo gets the owner information.
+// And extracting it into a function can improve the test coverage.
```
Comment (shenli) [inline ddl/owner_manager.go]: "And extracting it into a function can improve the test coverage."  This is not a proper comment.

**Comment by zimulala** [comment]
PTAL @shenli

**Comment by shenli** [comment]
LGTM

**Pull request merged**